app_server/app/util/Utility.js

```diff
@@ -60,3 +61,62 @@ exports.createLogger = function (filename) {
     ]
   });
 };
+
+var clearUserProfile =
```

Comment:
Yeap, in a stream object, I have to embed a user object (owner) into them. And to use it to clear the user credentials.